curation_concerns-models/app/jobs/upload_set_update_job.rb

```diff
@@ -26,20 +26,8 @@ def perform(login, upload_set_id, title, file_attributes, visibility)
     upload_set.update(status: ["Complete"])
 
     if denied.empty?
-      unless saved.empty?
```

Comment:
Ah, good catch, I didn't see that these are the descendants of `send_user_success_message` and `send_user_failure_message`